test/mtevbusted-script.in

```diff
@@ -1,4 +1,4 @@
-#!@bindir@/luamtev -L+@LUAROCKS_PATH@
+#!/opt/circonus/bin/luamtev -L+/opt/circonus/share/lua/5.1/?.lua;/opt/circonus/share/lua/5.1/?/init.lua
```

Comment:
Why are you hardcoding the autoconf variables here? I don't think this should be modified.

Reply:
my bad, didn't see that diff when creating the branch, its not supposed to have that change (was only needed for testing , where the bindir and luarocks_path have already been replaced in the install folder as "/opt/circonus/bin/mtevbusted"